modules/altinnsalary/model/altinnreport.class.php

```diff
@@ -588,8 +588,10 @@ function queryStringForCurrentlyEmployedEmployees() {
  */
   function queryStringForIncludedWorkRelations() {
     $query_work_relations = "SELECT wr.*
-                             FROM workrelation wr
-                             WHERE wr.WorkRelationID IN (" . implode($this->work_relation_ids, ', ') . ")";
+                             FROM workrelation wr";
+    if (!empty($this->work_relation_ids)) {
+      $query_work_relations .= " WHERE wr.WorkRelationID IN (" . implode($this->work_relation_ids, ', ') . ")";
```

Comment:
Should we return a query that will select all WR if  `$this->work_relation_ids` is not set?